bundle/edu.gemini.spdb.reports.collection/src/main/scala/edu/gemini/spdb/cron/osgi/Activator.scala

```diff
@@ -37,7 +38,8 @@ class Activator extends BundleActivator {
          "odbState"       -> OdbStateAgent.run,
          "odbMail"        -> OdbMailAgent.run,
          "weather"        -> new WeatherUpdater(c).run,
-         "archive"        -> Archiver.run(c))
+         "archive"        -> Archiver.run(c),
```

Comment:
Correct, it is an actual system cron job that makes an http request.  The frequency is just defined in the standard crontab.